sansa-query-spark/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,23 @@
+# Root logger option
```

Comment:
Do you really need this log configuration file?

Reply:
No problem, you can remove it.